R/xyplot.hydromad.R

```diff
@@ -127,7 +206,7 @@ qqmath.hydromad <-
       mod = fitted(x, all = all)
     )
     ## keep only common (corresponding) values
-    coredata(tsdat)[complete.cases(tsdat) == FALSE, ] <- NA
+    zoo::coredata(tsdat)[complete.cases(tsdat) == FALSE, ] <- NA
```

Comment:
zoo is listed as depends and is extensively used throughout hydromad. I wouldn't have thought the zoo:: prefix was needed here?

Reply:
I think you're right about this. I might have added `zoo::` at an earlier stage to explicitly call the function, but solved it better later on with `@importFrom zoo coredata`. I removed `zoo::` locally and ran `devtools::check` and it seemed to be fine. It will appear with the next commit.